packages/dashboard/src/portal/components/VotePopup.vue

```diff
@@ -44,6 +55,13 @@ export default class VotePopup extends Vue {
     this.openProposalDialog = true;
   }
 
+  async getUserNodes(farmIDs: any[]): Promise<any> {
+    const url = `${config.gridproxyUrl}/nodes?ret_count=true&page=1&size=10&farm_ids=${farmIDs}`;
```

Comment:
the farmIDs here in the query are a list while the gridproxy is waiting for the farmIDs just numbers separated by a comma.